internal/services/mssql/mssql_database_resource.go

```diff
@@ -1389,11 +1413,9 @@ func resourceMsSqlDatabaseSchema() map[string]*pluginsdk.Schema {
 					"email_account_admins": {
```

Comment:
this is a boolean in the SDK, so we should add a TODO to update this in 4.0:

```suggestion
					// NOTE: this is a Boolean in SDK rather than a String
					// TODO: update this to be `email_account_admins_enabled` in 4.0
					 "email_account_admins": {
```

Reply:
Fixed.